examples/activation/flake.lock

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Do we need to add the flake lock? I left them out so that it's always generated upon first use, since otherwise the lock file and the main repository can get out of sync.

Reply:
This is probably also where the CI error `error: cannot fetch input 'path:../..?lastModified=0&narHash=sha256-l%2BJhyv%2BZH/SapwRSnz9ye1pCd74u1S/4e3fbZai0dFs%3D' because it uses a relative path` is coming from.

Reply:
great points 🙏, I've removed the lock and updated the rest of the examples in the latest commit and CI looks much better!